AlfrescoContent/AlfrescoContent/Classes/Swaggers/Process/ProcessAPI.swift

```diff
@@ -155,6 +155,20 @@ open class ProcessAPI: NSObject {
         return requestBuilder.init(method: "POST", URLString: (URLString), parameters: parameters, isBody: true)
     }
     
+    open class func startWorkFlowProcess(params: StartWorkFlowBodyCreate, withCallback completion: @escaping ((_ data: Process?,_ error: Error?) -> Void)) {
```

Comment:
Either modify the existing code or delete the old one if a new class is created because the existing code will not longer be used.